File: CS/E1267/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraGrid.Bookmarks;

namespace E1267 {
    public partial class FormMain : Form {
        public FormMain() {
            InitializeComponent();
            new GridViewBookmarks(this.gridView1, "OrderID");
        }

        private void FormMain_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'nwindDataSet.Orders' table. You can move, or remove it, as needed.
            this.ordersTableAdapter.Fill(this.nwindDataSet.Orders);

        }
    }
}

File: VB/E1267/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Bookmarks

Namespace E1267
	Partial Public Class FormMain
		Inherits Form
		Public Sub New()
			InitializeComponent()
			Dim TempGridViewBookmarks As GridViewBookmarks = New GridViewBookmarks(Me.gridView1, "OrderID")
		End Sub

		Private Sub FormMain_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'nwindDataSet.Orders' table. You can move, or remove it, as needed.
			Me.ordersTableAdapter.Fill(Me.nwindDataSet.Orders)

		End Sub
	End Class
End Namespace